policies/templates/gcp_storage_bucket_retention_v1.yaml

```diff
@@ -103,7 +103,7 @@ spec:
         	violations := get_diff
         	count(violations) > 0
         	violation_msg := concat(" & ", violations)
-        	message := sprintf("Storage bucket %v has a retention policy violation: %v", [asset.name, violation_msg])
+        	message := sprintf("Storage bucket has a retention policy violation: %v", [asset.name, violation_msg])
```

Comment:
@rmaloloyon lets not change these values?

Reply:
Yes sure @leylmordor, but the expected output could be something like this. Try to revert it to the original constraints
```
 //storage.googleapis.com/mv-dev-ricardo-stateless-assets-gcs does not have
 'Storage bucket //storage.googleapis.com/mv-dev-ricardo-stateless-assets-gcs
```
Not sure on how to increase the limit while displaying the output here in GUI.

Reply:
I modified it a little bit with the template and I got these results, 
`---------------------------------
Constraints Diagnostic Analysis :
---------------------------------

 Does not have bucket policy only enabled - //storage.googleapis.com/mv-dev-ricardo-stateless-assets-gcs
 It has a retention policy violation - Storage bucket //storage.googleapis.com/mv-dev-ricardo-stateless-assets-gcs

Policy: 17 test, 15 passed, 0 warning, 2 failure, 0 exceptions`